### PR TITLE
added December What's New

### PR DESCRIPTION
**Pull request by mportman12**

### CODE REVIEW

**Comment by mportman12** [comment]
@bergerhoffer please review

File: supplementary_style_guide/introduction/whats-new.adoc [hunk 1]
```diff
@@ -29,6 +29,19 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 // .Other updates
 // * TODO
 
+[[december-2023]]
+== December 2023
+.Glossary entries
+* xref:ibm-cloud[IBM Cloud®]: Added glossary entry
+* xref:now[now]: Updated glossary entry to "use with caution"
```
Comment (bergerhoffer) [inline supplementary_style_guide/introduction/whats-new.adoc:36]: This one is out of alphabetical order

File: supplementary_style_guide/introduction/whats-new.adoc [hunk 2]
```diff
@@ -29,6 +29,19 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
...
+* look up: Removed glossary entry because guidance already exists in the _Merriam-Webster Dictionary_
+* look-up: Removed glossary entry because guidance already exists in the _Merriam-Webster Dictionary_
+* lookup: Removed glossary entry because guidance already exists in the _Merriam-Webster Dictionary_
+* xref:previously[previously]: Added glossary entry
+
+.Style guidance
+* xref:https://redhat-documentation.github.io/supplementary-style-guide/#ip-addresses-and-mac-addresses[IP addresses and MAC addresses]: Added examples of reserved IP addresses and MAC addresses
```
Comment (bergerhoffer) [inline supplementary_style_guide/introduction/whats-new.adoc:43]: This xref isn't correct

```suggestion
* xref:ip-addresses-and-mac-addresses[IP addresses and MAC addresses]: Added examples of reserved IP addresses and MAC addresses
```

**Review by bergerhoffer — COMMENTED** [review]
Two things!

File: supplementary_style_guide/introduction/whats-new.adoc (hunk 1 above)
Comment (mportman12) [reply supplementary_style_guide/introduction/whats-new.adoc]: Fixed

File: supplementary_style_guide/introduction/whats-new.adoc (hunk 2 above)
Comment (mportman12) [reply supplementary_style_guide/introduction/whats-new.adoc]: Fixed

**Comment by mportman12** [comment]
@bergerhoffer Made your fixes, please review again

**Review by bergerhoffer — APPROVED** [review]
@mportman12 the actual update lgtm, but the vale check is failing. Not sure what is going on with that - CC @aireilly / @bburt-rh

**Comment by aireilly** [comment]
Oh there is a breaking failure with the latest version of Vale. Will submit a fix presently.

**Comment by bergerhoffer** [comment]
> Oh there is a breaking failure with the latest version of Vale. Will submit a fix presently.

Thanks @aireilly!

**Comment by aireilly** [comment]
@bergerhoffer PTAL and merge: https://github.com/redhat-documentation/supplementary-style-guide/pull/435

This PR fixes the Vale issue.

**Comment by mportman12** [comment]
rebuild

**Pull request merged**